<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
   <addaction name="actionCreatePNG"/>
   <addaction name="actionPreferences"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionZoomout"/>
   <addaction name="actionZoomin"/>
   <addaction name="actionNormalSize"/>
   <addaction name="actionFitWindow"/>
   <addaction name="actionFitWindowWidth"/>
   <addaction name="separator"/>
   <addaction name="actionWhatsthis"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSatSelection"/>
   <addaction name="actionMeteosat"/>
   <addaction name="actionCylindricalEquidistant"/>
   <addaction name="action3DGlobe"/>
   <addaction name="actionImage"/>
   <addaction name="actionShowToolbox"/>
  </widget>
  <action name="actionSatSelection">
   <property name="icon">
    <iconset resource="EUMETCastView.qrc">
     <normaloff>:/icons/satellite.jpg</normaloff>:/icons/satellite.jpg</iconset>
   </property>
   <property name="text">
    <string>Satellite Setup</string>
   </property>
   <property name="toolTip">
    <string>Satellite Setup</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="EUMETCastView.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Quit the program</string>
   </property>
  </action>
  <action name="actionCreatePNG">
   <property name="icon">
    <iconset resource="EUMETCastView.qrc">
     <normaloff>:/icons/camera.png</normaloff>:/icons/camera.png</iconset>
   </property>
   <property name="text">
    <string>Create PNG file</string>
   </property>
   <property name="toolTip">
    <string>Create a JPG/PNG image file</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="EUMETCastView.qrc">
     <normaloff>:/icons/Preferences_48x48.png</normaloff>:/icons/Preferences_48x48.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionCylindricalEquidistant">
   <property name="icon">
    <iconset resource="EUMETCastView.qrc">
     <normaloff>:/icons/Equi_projection_48x48.png</normaloff>:/icons/Equi_projection_48x48.png</iconset>
   </property>
   <property name="text">
    <string>Cylindrical Equidistant</string>
   </property>
   <property name="toolTip">
    <string>Cylindrical Equidistant</string>
   </property>
  </action>
  <action name="action3DGlobe">
   <property name="icon">
    <iconset resource="EUMETCastView.qrc">
     <normaloff>:/icons/300px-Orthographic_projection_SW.png</normaloff>:/icons/300px-Orthographic_projection_SW.png</iconset>
   </property>
   <property name="text">
    <string>3D Globe</string>
   </property>
   <property name="toolTip">
    <string>3D Globe</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="icon">
    <iconset resource="EUMETCastView.qrc">
     <normaloff>:/icons/image.png</normaloff>:/icons/image.png</iconset>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
   <property name="toolTip">
    <string>Image</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="EUMETCastView.qrc">
     <normaloff>:/icons/questionmark.png</normaloff>:/icons/questionmark.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionShowToolbox">
   <property name="icon">
    <iconset resource="EUMETCastView.qrc">
     <normaloff>:/icons/options.png</normaloff>:/icons/options.png</iconset>
   </property>
   <property name="text">
    <string>Show Toolbox</string>
   </property>
   <property name="toolTip">
    <string>Toggle Toolbox</string>
   </property>
  </action>
  <action name="actionMeteosat">
   <property name="icon">
    <iconset resource="EUMETCastView.qrc">
     <normaloff>:/icons/satell2.png</normaloff>:/icons/satell2.png</iconset>
   </property>
   <property name="text">
    <string>Meteosat</string>
   </property>
   <property name="toolTip">
    <string>Geostationary satellites</string>
   </property>
  </action>
  <action name="actionNormalSize">
   <property name="icon">
    <iconset resource="EUMETCastView.qrc">
     <normaloff>:/icons/zoom-original.png</normaloff>:/icons/zoom-original.png</iconset>
   </property>
   <property name="text">
    <string>Normal Size</string>
   </property>
   <property name="toolTip">
    <string>Normal Size</string>
   </property>
  </action>
  <action name="actionFitWindow">
   <property name="icon">
    <iconset resource="EUMETCastView.qrc">
     <normaloff>:/icons/zoom-fit-best.png</normaloff>:/icons/zoom-fit-best.png</iconset>
   </property>
   <property name="text">
    <string>Fit Window</string>
   </property>
   <property name="toolTip">
    <string>Fit Window</string>
   </property>
  </action>
  <action name="actionZoomin">
   <property name="icon">
    <iconset resource="EUMETCastView.qrc">
     <normaloff>:/icons/zoom-in.png</normaloff>:/icons/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoomout">
   <property name="icon">
    <iconset resource="EUMETCastView.qrc">
     <normaloff>:/icons/zoom-out.png</normaloff>:/icons/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionFitWindowWidth">
   <property name="icon">
    <iconset resource="EUMETCastView.qrc">
     <normaloff>:/icons/zoom-fit-best-width.png</normaloff>:/icons/zoom-fit-best-width.png</iconset>
   </property>
   <property name="text">
    <string>Fit Window Width</string>
   </property>
   <property name="toolTip">
    <string>Fit Window Width</string>
   </property>
  </action>
  <action name="actionWhatsthis">
   <property name="icon">
    <iconset resource="EUMETCastView.qrc">
     <normaloff>:/icons/question-mark-cursor_318-113947.png</normaloff>:/icons/question-mark-cursor_318-113947.png</iconset>
   </property>
   <property name="text">
    <string>Whats this ?</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EUMETCastView.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showPreferences()</slot>
 </slots>
</ui>
